app/src/test/java/nerd/tuxmobil/fahrplan/congress/validation/MetaValidationTest.kt

```diff
@@ -0,0 +1,31 @@
+package nerd.tuxmobil.fahrplan.congress.validation
+
+import com.google.common.truth.Truth.assertThat
+import info.metadude.android.eventfahrplan.network.models.Meta
+import nerd.tuxmobil.fahrplan.congress.validation.MetaValidation.validate
+import org.junit.Test
+
+class MetaValidationTest {
+
+    @Test
+    fun `validate returns timeZoneName = null if timeZoneName = null`() {
+        assertThat(Meta().validate()).isEqualTo(Meta())
```

Comment:
You're making untested assumptions about the default value of `timeZoneName`. I suggest either adding a test to check that assumption (`assertThat(Meta().timeZoneName).isNull()`) or explicitly setting `timeZoneName` (`Meta(timeZoneName = null)`).

Reply:
True. Thanks. Revised in 0714fdd0771a25652917a74919f1ab435a3da221.

Reply:
The test is called "validate returns timeZoneName = default value if timeZoneName = default value" but it's not specifically checking `timeZoneName`. Maybe something along the lines of `instance with default values is valid`?

Reply:
Changed in d484208f08b71d3bd6a14af4b35356b639df0013.